Xamarin.Forms.DualScreen/DualScreenService.android.cs

```diff
@@ -80,26 +80,47 @@ public static void Init(Activity activity)
 
 				if (_HingeService._hingeSensor != null)
 				{
-					_HingeService._hingeSensor.OnSensorChanged -= _HingeService.OnSensorChanged;
-					_HingeService._hingeSensor.StopListening();
+					//_HingeService._hingeSensor.OnSensorChanged -= _HingeService.OnSensorChanged;
```

Comment:
Is it causing some kind of problems?, can the refresh rate be reduced perhaps? (SensorDelay).

Reply:
Yea @Redth is going to look into that

I wonder if this is something we should expose cross platform as well.. Turning listening off and on so it's just not firing off by default.

For now I'm just going to add a getter for the android dll so you can retrieve the sensor